content/en/platform/corda/5.0-dev-preview-2/getting-started/prerequisites/prerequisites.md

```diff
@@ -0,0 +1,62 @@
+---
+date: '2022-09-19'
+title: "Third-party prerequisites"
+menu:
+  corda-5-dev-preview2:
+    parent: corda-5-dev-preview-start
+    identifier: corda-5-dev-preview-prereqs
+    weight: 1000
+section_menu: corda-5-dev-preview2
+---
+
+## Software prerequisites
+
+Corda 5 DP 2 has been tested with the following:
+
+| Software      | Version |
+| ----------- | ----------- |
+| Operating systems      | <li>Mac OS (intel and ARM)</li><li>Windows 10/11</li><li>Linux</li>     |
+| Java   | Azul Zulu JDK 11 (Other versions should work but have not been extensively tested.)  |
+| Intellij    | ~v2021.X.Y community edition   |
+| git | ~v2.24.1    |
+| Docker | Docker Engine ~v20.X.Y or Docker Desktop ~v3.5.X    |
+<!--| Gradle |  7.0+   |-->
+
+
+If you want to experiment with multi-worker cluster deployments, you will also need:
+
+* Kubernetes (incl. kubectl)
+* Helm
+
+However, Developer Preview 2 focuses on the developer rather than operator experience, so [Kubernetes deployments](../deploying/local-deployment.md) are not required to use Developer Preview 2.
```

Comment:
```suggestion
However, Developer Preview 2 focuses on the developer rather than operator experience, so Kubernetes deployments are not required to use Developer Preview 2.
```